### PR TITLE
Fix: [i18n] Some strings in admin-apps-page.php are not displayed translated

### PR DESCRIPTION
**Pull request by DAnn2012**
## PR Checklist

- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* [i18n] Some strings in admin-apps-page.php are not displayed translated

## Description
An explanation of what is done in this PR

* Fix for some strings in admin-apps-page.php are not displayed translated if the language is not English

## Test instructions
This PR can be tested by following these steps:

* Before (Language set to Spanish)
before

* After (Language set to Spanish)
after

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by blackcolt — DISMISSED** [review]

File: modules/apps/admin-apps-page.php [hunk 1]
```diff
@@ -93,18 +93,18 @@ private static function filter_wporg_app( $app ) {
 
 		if ( static::is_plugin_installed( $app['file_path'] ) ) {
 			if ( current_user_can( 'activate_plugins' ) ) {
-				$app['action_label'] = 'Activate';
+				$app['action_label'] = __( 'Activate', 'elementor' );
```
Comment (blackcolt) [inline modules/apps/admin-apps-page.php:96]: This needs to be esc_html__ and not just __
To avoid HTML injections in translations.

Comment (blackcolt) [reply modules/apps/admin-apps-page.php:96]:
```suggestion
				$app['action_label'] = esc_html__( 'Activate', 'elementor' );
```

File: modules/apps/admin-apps-page.php [hunk 2]
```diff
@@ -93,18 +93,18 @@ private static function filter_wporg_app( $app ) {
...
 				$app['action_url'] = static::get_activate_plugin_url( $app['file_path'] );
 			} else {
-				$app['action_label'] = 'Cannot Activate';
+				$app['action_label'] = __( 'Cannot Activate', 'elementor' );
```
Comment (blackcolt) [inline modules/apps/admin-apps-page.php:99]:
```suggestion
				$app['action_label'] = esc_html__( 'Cannot Activate', 'elementor' );
```

File: modules/apps/admin-apps-page.php [hunk 3]
```diff
@@ -93,18 +93,18 @@ private static function filter_wporg_app( $app ) {
...
 				$app['action_url'] = '#';
 			}
 		} else {
 			if ( current_user_can( 'install_plugins' ) ) {
-				$app['action_label'] = 'Install';
+				$app['action_label'] = __( 'Install', 'elementor' );
```
Comment (blackcolt) [inline modules/apps/admin-apps-page.php:104]:
```suggestion
				$app['action_label'] = esc_html__( 'Install', 'elementor' );
```

File: modules/apps/admin-apps-page.php [hunk 4]
```diff
@@ -93,18 +93,18 @@ private static function filter_wporg_app( $app ) {
...
 				$app['action_url'] = static::get_install_plugin_url( $app['file_path'] );
 			} else {
-				$app['action_label'] = 'Cannot Install';
+				$app['action_label'] = __( 'Cannot Install', 'elementor' );
```
Comment (blackcolt) [inline modules/apps/admin-apps-page.php:107]:
```suggestion
				$app['action_label'] = esc_html__( 'Cannot Install', 'elementor' );
```

File: modules/apps/admin-apps-page.php [hunk 5]
```diff
@@ -126,10 +126,10 @@ private static function filter_ecom_app( $app ) {
 		}
 
 		if ( current_user_can( 'activate_plugins' ) ) {
-			$app['action_label'] = 'Activate';
+			$app['action_label'] = __( 'Activate', 'elementor' );
```
Comment (blackcolt) [inline modules/apps/admin-apps-page.php:129]:
```suggestion
			$app['action_label'] = esc_html__( 'Activate', 'elementor' );
```

File: modules/apps/admin-apps-page.php [hunk 6]
```diff
@@ -126,10 +126,10 @@ private static function filter_ecom_app( $app ) {
...
 			$app['action_url'] = static::get_activate_plugin_url( $app['file_path'] );
 		} else {
-			$app['action_label'] = 'Cannot Activate';
+			$app['action_label'] = __( 'Cannot Activate', 'elementor' );
```
Comment (blackcolt) [inline modules/apps/admin-apps-page.php:132]:
```suggestion
			$app['action_label'] = esc_html__( 'Cannot Activate', 'elementor' );
```

**Review by ManorHazaz — DISMISSED** [review]
Fix blackcolt's comments please

**Comment by DAnn2012** [comment]
Hello @blackcolt and @ManorHazaz 

I made the changes requested by the reviewer.

Thanks

**Review by hein-obox — DISMISSED** [review]

**Review by hein-obox — APPROVED** [review]

File: modules/apps/admin-apps-page.php [hunk 7]
```diff
@@ -108,18 +108,18 @@ private static function filter_wporg_app( $app ) {
 
 		if ( self::$plugin_status_adapter->is_plugin_installed( $app['file_path'] ) ) {
 			if ( current_user_can( 'activate_plugins' ) ) {
-				$app['action_label'] = 'Activate';
+				$app['action_label'] = esc_html__( 'Activate', 'elementor' );
```
Comment (hein-obox) [inline modules/apps/admin-apps-page.php:111]: @blackcolt Can you please review?

**Comment by DAnn2012** [comment]
Hi @louiswol94 ,

please could you kindly take a look at my PR?

Thank you.

**Comment by DAnn2012** [comment]
Hi @KingYes  ,

please could you kindly take a look at my PR?

Thank you.

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**